Assets/Scripts/ZoomSystem/DetailTextureImage.cs

```diff
@@ -0,0 +1,44 @@
+using UnityEngine;
+
+/**
+ * Controls the visibility of a texture depending on a Scanner's
+ * zoom level.
+ * TODO: Has a dependency on Scanner type
+ */
+[RequireComponent(typeof(Renderer))]
+public class DetailTextureImage : MonoBehaviour
+{
+    [SerializeField] Scanner scannedBy;
+    [SerializeField] float minOpacity = 0f;
+    [SerializeField] float maxOpacity = 0.75f;
+    [SerializeField] float zoomLevelForMinOpacity = 1f;
+    [SerializeField] float zoomLevelForMaxOpacity = 5f;
+
+
+    void OnZoomChanged(float zoomLevel)
+    {
+        Material mat = GetComponent<Renderer>().material;
+        Color matColor = mat.color;
+        matColor.a = minOpacity +
+            (Mathf.Sqrt(Mathf.Clamp(
+                ((zoomLevel - zoomLevelForMinOpacity) / zoomLevelForMaxOpacity),
+                0f, 1f)) * maxOpacity) - minOpacity;
+        mat.color = matColor;
+    }
+
+    void Awake()
```

Comment:
I tend to prefer OnEnable and OnDisable as they are run every time the code recompiles while in play mode so that things continue to work. Not a huge deal in a jam, but mentioning in case there is a reason for your choice that I don't know :D

Reply:
Good idea, I'll push another commit.